python/chilldkg_ref/chilldkg.py

```diff
@@ -600,14 +589,14 @@ def participant_finalize(
     return dkg_output, RecoveryData(eq_input + cmsg2.cert)
 
 
-def participant_blame(
-    blame_state: ParticipantBlameState,
-    cblame: CoordinatorBlameMsg,
+def participant_investigate(
+    error: UnknownFaultyParticipantOrCoordinatorError,
+    cinvestigation: CoordinatorInvestigationMsg,
```

Comment:
I think we can use a short form "inv" in *variable names* (everywhere):
```suggestion
    cinv: CoordinatorInvestigationMsg,
```

Reply:
Fixed